UndertaleModTool/Editors/UndertaleSpriteEditor.xaml.cs

```diff
@@ -112,7 +112,7 @@ private void ExportAll_Click(object sender, RoutedEventArgs e)
                             }
                             catch (Exception ex)
                             {
-                                MessageBox.Show("Failed to export file: " + ex.Message, "Failed to export file", MessageBoxButton.OK, MessageBoxImage.Error);
+                                MessageBox.Show(Application.Current.MainWindow as MainWindow, "Failed to export file: " + ex.Message, "Failed to export file", MessageBoxButton.OK, MessageBoxImage.Error);
```

Comment:
Can you create a property for this, so it doesn't have to get copypasted to every catch?